script/sdbSpecList.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+#
+# This shell script prints tizen device serial name on each line.
+#
+# on `sdb devices`, it will show result below.
+#
+# List of devices attached 
+# 192.168.223.1:26101 	device    	0
+# 192.168.224.1:26101 	device    	0
+#
+# so, only imformation we need is list of -s in first column from second row to end.
```

Comment:
Could you explain what `-s` is?

Reply:
um... I think I make some mistake on this comment. I will fix it. Sorry for bothering.